store/sql_user_store.go

```diff
@@ -50,6 +50,7 @@ func NewSqlUserStore(sqlStore *SqlStore) UserStore {
 		table.ColMap("NotifyProps").SetMaxSize(2000)
 		table.ColMap("Locale").SetMaxSize(5)
 		table.ColMap("MfaSecret").SetMaxSize(128)
+		table.ColMap("Position").SetMaxSize(35)
```

Comment:
OK, will change this to 64 char hard limit in DB, with validation/UI limit of 35 char.